prow/gerrit/client/client.go

```diff
@@ -179,14 +179,26 @@ func (l LastSyncState) DeepCopy() LastSyncState {
 	return result
 }
 
+type roundTripperWithHeader struct {
+	upstream http.RoundTripper
+}
+
+func (rt *roundTripperWithHeader) RoundTrip(r *http.Request) (*http.Response, error) {
+	r.Header.Add("user-agent", "prow")
```

Comment:
We should probably include the prow component name since multiple components may connect to the gerrit API.
https://github.com/kubernetes/test-infra/blob/f193539e9f83b9671be3134d8d82554161bda0de/prow/version/doc.go#L28
```suggestion
	r.Header.Add("user-agent", "prow" + version.Name)
```

Reply:
good point, added both.
- "prow" for Gerrit manager
- "prow/<COMPONENT>" for further investigation